ui/app/components/job-row.js

```diff
@@ -12,6 +12,10 @@ export default class JobRow extends Component {
 
   job = null;
 
+  // One of independent, parent, or child. Used to customize the template
+  // based on the relationship of this job to others.
+  context = 'independent';
```

Comment:
I couldn’t find a use of `parent` for this, did you change your mind on that?

Reply:
This was a note for future us for where I was going with this. These are the three types of contexts I was imagining when I stuck this field in here.

The fact it needs an explanation may mean the attr name isn't good. But it's also consistent with AllocationRow.

Reply:
oh yes, makes sense. Well I don’t have a better name suggestion and the comment helps contextualise 😆

Reply:
I'm gonna merge it then 🙂